### PR TITLE
ACCUMULO-4189 - Added direct links to documentation on Accumulo website

### PR DESCRIPTION
**Pull request by mikewalch**
#91 should be reviewed/merged first before this PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,5 +106,10 @@ with Accumulo 1.6.2 and 1.7.0 all API changes will follow [semver 2.0][12]
 [7]: http://thrift.apache.org/
 [8]: http://accumulo.apache.org/notable_features.html
 [9]: http://maven.apache.org/
+[10]: http://accumulo.apache.org/user_manual.html
```
Comment (ctubbsii) [inline README.md]: These should use https wherever possible. I know some of the older links don't have it, but they should. At the very least, new ones should.

**Comment by mikewalch** [comment]
In 17288b37bab2b42, I fixed the README to point to latest pages that redirect to current version.  I also made the links to the Accumulo website use https.

File: README.md [hunk 2]
```diff
@@ -101,14 +97,19 @@ The Accumulo project maintains binary compatibility across this API within a
...
 [3]: TESTING.md
 [4]: http://research.google.com/archive/bigtable.html
 [5]: http://hadoop.apache.org
 [6]: http://zookeeper.apache.org
 [7]: http://thrift.apache.org/
-[8]: http://accumulo.apache.org/notable_features.html
+[8]: https://accumulo.apache.org/notable_features.html
 [9]: http://maven.apache.org/
+[10]: https://accumulo.apache.org/user-manual/latest/
+[11]: https://accumulo.apache.org/examples/latest/
```
Comment (ctubbsii) [inline README.md]: [nit] Adds trailing whitespace. (some above, also)

Comment (mikewalch) [inline README.md]: Removed trailing whitespace in b3aad3d96

**Pull request merged**